plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -888,47 +888,34 @@ class producer_plugin_impl : public std::enable_shared_from_this<producer_plugin
       _time_tracker.clear();
    }
 
-   bool on_incoming_block(const signed_block_ptr& block, const block_id_type& id, const block_handle& bh) {
+   bool on_incoming_block() {
```

Comment:
Should this be renamed `on_any_incoming_blocks` or add a comment saying the function does not process a specific block.

Reply:
Added a comment.